README.rst

```diff
@@ -307,6 +307,32 @@ Here is an example settings fragment for `django-redis`:
         },
     }
 
+
+Suspending and resuming workers
```

Comment:
Can we title acse this?

Reply:
Done. BTW you might consider adding a YAML linter to enforce this kind of stuff automatically.